### PR TITLE
Add `topics` to package intl `pubspec.yaml`

### PR DESCRIPTION
**Pull request by szakarias**
Topics help users on pub.dev discover packages and related packages.

You can see a list of [topics here](https://pub.dev/topics).
You can also make your own topics.

Topics are added to pubspec.yaml as follows:

```topics:
 - my-topic
 - some-other-topic
```

See also: https://dart.dev/tools/pub/pubspec#topics

### CODE REVIEW

**Comment by szakarias** [comment]
@mosuem  Do you know whether this changed should be applied in Google3 first?

**Comment by mosuem** [comment]
Yes, PRs here are merged automatically by copybara after being approved in g3.

**Review by mosuem — APPROVED** [review]

File: pkgs/intl/pubspec.yaml [hunk 1]
```diff
@@ -6,6 +6,10 @@ description: >-
   internationalization issues.
 repository: https://github.com/dart-lang/i18n/tree/main/pkgs/intl
 
+topics:
+ - i18n
+ - intl
```
Comment (kevmoo) [inline pkgs/intl/pubspec.yaml:11]: Maybe add `internationalization` too? Or what's common on pub?

**Comment by szakarias** [comment]
Moved to g3

File: pkgs/intl/pubspec.yaml (hunk 1 above)
Comment (szakarias) [reply pkgs/intl/pubspec.yaml:11]: We have a file for canonical topics. Added them as aliases there.

**Pull request closed without merging**